include/blasfeo_d_aux.h

```diff
@@ -33,12 +33,18 @@
  *
  */
 
+#include "blasfeo_common.h"
+
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#ifndef BLASFEO_D_AUX
+#define BLASFEO_D_AUX
 
 
+#include "blasfeo_common.h"
```

Comment:
again, why are you including the headers twice?

Reply:
fixed